eth/api.go

```diff
@@ -637,17 +637,40 @@ func storageRangeAt(st state.Trie, start []byte, maxResult int) StorageRangeResu
 	return result
 }
 
-func (api *PrivateDebugAPI) GetDirtyAccountsByNumber(startNum, endNum uint64) ([]common.Address, error) {
-	return api.getDirtyAccounts(api.eth.blockchain.GetBlockByNumber(startNum), api.eth.blockchain.GetBlockByNumber(endNum))
+// GetModifiedAccountsByumber returns all accounts that have changed between the
+// two blocks specified. A change is defined as a difference in nonce, balance,
+// code hash, or storage hash.
+// With one parameter, returns the list of accounts modified in the specified block.
+func (api *PrivateDebugAPI) GetModifiedAccountsByNumber(startNum, endNum *uint64) ([]common.Address, error) {
+	// With one argument, compare x-1 with x
+	if endNum == nil {
+		return api.getModifiedAccounts(
+			api.eth.blockchain.GetBlockByNumber(*startNum - 1),
```

Comment:
If startnum is the genesis, this will get funky :) Perhaps a special handling is needed for that?

Reply:
``` 
> debug.getModifiedAccountsByNumber(0)
Error: Both start block and end block are required
    at web3.js:3143:20
    at web3.js:6347:15
```